featuretools/primitives/transform_primitive.py

```diff
@@ -336,6 +337,30 @@ class Weekday(DatetimeUnitBasePrimitive):
     name = "weekday"
 
 
+class NumCharacter(TransformPrimitive):
+    """
+    Return the characters in a given string.
+    """
+    name = 'characters'
+    input_types = [Text]
+    return_type = Numeric
+
+    def get_function(self):
+        return lambda array: pd.Series([len(x) for x in array])
```

Comment:
I think you can use more pandas built syntax for this since the `array` variable is going to be a pandas series (please double check this though)

```
In [1]: array = pd.Series(["1","12 2","1212 3"])

In [2]: array.str.len()
Out[2]: 
0    1
1    4
2    6
dtype: int64
```

so, it would just be
```python
def get_function(self):
    return lambda array: array.str.len()
```

Reply:
Array is a `numpy.ndarray`

Reply:
thanks for double checking.  I don't have strong preference, but perhaps do this to avoid the list comprehension

```python 
pd.Series(array).str.len()
```